File: app/src/main/java/com/yelai/wearable/model/ExaminationScore.java
package com.yelai.wearable.model;

import java.io.Serializable;

import kotlin.jvm.Transient;

/**
 * Created by hr on 18/11/12.
 */

public class ExaminationScore implements Serializable {

    private String memberId;
    private String name;
    private String headImg;
    private String timeStr;
    private String time;
    private String height;


    @Transient
    private String rank;

    public String getRank() {
        return rank;
    }

    public void setRank(String rank) {
        this.rank = rank;
    }

    public String getMemberId() {
        return memberId;
    }

    public void setMemberId(String memberId) {
        this.memberId = memberId;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getHeadImg() {
        return headImg;
    }

    public void setHeadImg(String headImg) {
        this.headImg = headImg;
    }

    public String getTimeStr() {
        return timeStr;
    }

    public void setTimeStr(String timeStr) {
        this.timeStr = timeStr;
    }

    public String getTime() {
        return time;
    }

    public void setTime(String time) {
        this.time = time;
    }

    public String getHeight() {
        return height;
    }

    public void setHeight(String height) {
        this.height = height;
    }

    public String getWeight() {
        return weight;
    }

    public void setWeight(String weight) {
        this.weight = weight;
    }

    public String getResult() {
        return result;
    }

    public void setResult(String result) {
        this.result = result;
    }

    private String weight;
    private String result;

//    @JsonAdapter(EmptyStringTypeAdapter::class) var memberId:String = ""
//
//    @JsonAdapter(EmptyStringTypeAdapter::class) var name:String = ""
//
//    @JsonAdapter(EmptyStringTypeAdapter::class) var headImg:String = ""
//
//    @JsonAdapter(EmptyStringTypeAdapter::class) var timeStr:String = ""
//
//    @JsonAdapter(EmptyStringTypeAdapter::class) var time:String = ""
//
//    @JsonAdapter(EmptyStringTypeAdapter::class) var height:String = ""
//
//    @JsonAdapter(EmptyStringTypeAdapter::class) var weight:String = ""
//
//    @JsonAdapter(EmptyStringTypeAdapter::class) var result:String = ""


}

File: app/src/main/java/com/yelai/wearable/model/ExaminationScore.kt
package com.yelai.wearable.model

import com.google.gson.annotations.JsonAdapter
import com.yelai.wearable.net.gson.EmptyStringTypeAdapter
import java.io.Serializable

/**
 * Created by hr on 18/11/11.
 * status： 0：未开考 1：考试中 2：考试完成
 * type : 1：身高体重 2：握力 3：引体向上 4：仰卧起坐 5：1000/800m跑步 6：50m跑步 7：肺活量 8：做立体前驱
 */

//class ExaminationScore : Serializable{
//
//    @JsonAdapter(EmptyStringTypeAdapter::class) var memberId:String = ""
//
//    @JsonAdapter(EmptyStringTypeAdapter::class) var name:String = ""
//
//    @JsonAdapter(EmptyStringTypeAdapter::class) var headImg:String = ""
//
//    @JsonAdapter(EmptyStringTypeAdapter::class) var timeStr:String = ""
//
//    @JsonAdapter(EmptyStringTypeAdapter::class) var time:String = ""
//
//    @JsonAdapter(EmptyStringTypeAdapter::class) var height:String = ""
//
//    @JsonAdapter(EmptyStringTypeAdapter::class) var weight:String = ""
//
//    @JsonAdapter(EmptyStringTypeAdapter::class) var result:String = ""
//
//}
